visualization/app/codeCharta/ui/ribbonBar/ribbonBar.component.html

```diff
@@ -1,106 +1,115 @@
-<cc-search-panel></cc-search-panel>
+<div class="ribbon-bar">
```

Comment:
do we need this extra wrapper? We could apply all styles of `.ribbon-bar` to the element `cc-ribbon-bar`, can we not?